hoomd/md/BendingRigidityMeshForceComputeGPU.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2009-2024 The Regents of the University of Michigan.
+// Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+#include "BendingRigidityMeshForceCompute.h"
+#include "BendingRigidityMeshForceComputeGPU.cuh"
+#include "hoomd/Autotuner.h"
+
+#include <memory>
+
+/*! \file BendingRigidityMeshForceComputeGPU.h
+    \brief Declares a class for computing bending rigidity energy forces on the GPU
+*/
+
+#ifdef __HIPCC__
+#error This header cannot be compiled by nvcc
+#endif
+
+#ifndef __BENDINGRIGIDITYMESHFORCECOMPUTE_GPU_H__
+#define __BENDINGRIGIDITYMESHFORCECOMPUTE_GPU_H__
+
+namespace hoomd
+    {
+namespace md
+    {
+
+//! Computes bending rigidity forces on the mesh on the GPU
+/*! Bending rigidity forces are computed on every particle in a mesh.
+
+    \ingroup computes
+*/
+class PYBIND11_EXPORT BendingRigidityMeshForceComputeGPU : public BendingRigidityMeshForceCompute
```

Comment:
```suggestion
class BendingRigidityMeshForceComputeGPU : public BendingRigidityMeshForceCompute
```
I don't believe the PYBIND11_EXPORT is necessary anymore.

Reply:
Why not? Did pybind11 change the default export policy?

Reply:
None of the pybind11 examples for classes on their [documentation page](https://pybind11.readthedocs.io/en/stable/classes.html#object-oriented-code) use this anymore.

Reply:
The most recent mention of export in the upgrade guide states that explicit exports are required when a shared library is linked by another:
https://pybind11.readthedocs.io/en/stable/upgrade.html?highlight=export#stricter-enforcement-of-hidden-symbol-visibility-for-pybind11-modules

This page in the docs confirm the requirement:
https://pybind11.readthedocs.io/en/stable/advanced/misc.html#partitioning-code-over-multiple-extension-modules

If you remove the `PYBIND11_EXPORT`, then I expect you will get linking errors. If not now, then when a user attempts to extend this library with a component.

An better solution would be to refactor the implementation and python bindings into separate `.so` files. Then `PYBIND11_EXPORT` would not be required. This would be a large porting effort with the extensive use of template generated classes in HOOMD.